lib/embed/viewer/common_viewer.rb

```diff
@@ -238,7 +238,7 @@ def header_title_logic
 
       def header_title_html(doc)
         doc.span(class: 'sul-embed-header-title') do
-          doc.text @purl_object.title
+          doc.text @purl_object.ng_xml.xpath('//dc:title','dc'=>'http://purl.org/dc/elements/1.1/').text
```

Comment:
Perhaps the change to the definition of what the PURL object's title is should be in the PURL object model: https://github.com/sul-dlss/sul-embed/blob/master/app/models/embed/purl.rb#L10-L12